revenuecat_examples/purchase_tester/lib/src/app.dart

```diff
@@ -147,7 +148,8 @@ class _UpsellScreenState extends State<UpsellScreen> {
 
         buttonThings.add(ElevatedButton(
           onPressed: () async {
-            await RevenueCatUI.presentPaywall();
+            final paywallResult = await RevenueCatUI.presentPaywall();
+            log('Paywall result: $paywallResult');
```

Comment:
Right now I'm only logging this. I did run into some issues getting this log to show for iOS, so I used the `dart:developer` log method for that which fixed the issue for me.